AmplifyPlugins/Geo/AWSLocationGeoPluginTests/AWSLocationGeoPluginClientBehaviorTests.swift

```diff
@@ -27,12 +27,12 @@ class AWSLocationGeoPluginClientBehaviorTests: AWSLocationGeoPluginTestBase {
     func testSearchForText() {
         let expResult = expectation(description: "Receive result")
 
-        geoPlugin.search(for: searchText) { [weak self] result in
+        geoPlugin.search(for: searchText, options: nil) { [weak self] result in
```

Comment:
This doesn't impact the PR, but we should discuss whether we want to maintain tests for our deprecated completion handler APIs across the board going forward.